node_modules/gh-users/lib/internal/dao.js

```diff
@@ -370,20 +370,26 @@ var getUserCalendar = module.exports.getUserCalendar = function(user, includeCon
         dateTimeFilter.end = {'lte': end};
     }
 
+    // This query fetches such a massive amount of data, that the CPU usage
+    // goes through the roof when Sequelize serialises it. To minimise this,
+    // we retrieve only those columns we really need
```

Comment:
Can we add a verification to the tests that ensure that no other data is returned?